File: backend/src/main/java/com/zzang/chongdae/comment/service/dto/CommentCreatedAtResponse.java
package com.zzang.chongdae.comment.service.dto;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.time.LocalTime;

public record CommentCreatedAtResponse(LocalDate date, LocalTime time) {

    public CommentCreatedAtResponse(LocalDateTime createdAt) {
        this(createdAt.toLocalDate(), createdAt.toLocalTime());
    }
}

File: android/app/src/main/java/com/zzang/chongdae/remote/dto/response/comment/CommentCreatedAtResponse.kt
package com.zzang.chongdae.remote.dto.response.comment

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class CommentCreatedAtResponse(
    @SerialName("date")
    val date: String,
    @SerialName("time")
    val time: String,
)
